src/Savegame/Soldier.cpp

```diff
@@ -160,61 +160,7 @@ void Soldier::load(const YAML::Node& node, const Mod *mod, SavedGame *save, cons
 	_training = node["training"].as<bool>(_training);
 	_returnToTrainingWhenHealed = node["returnToTrainingWhenHealed"].as<bool>(_returnToTrainingWhenHealed);
 
-	// This is here just to fix older saves, can be removed in year 2020 :)
```

Comment:
Only the 4 lines below are obsolete, not everything. I'll remove them when the time comes.